pyproject.toml

```diff
@@ -94,11 +91,7 @@ exasol-integration-test-docker-environment = "^3.1.0"
 pytest-history = ">=0.2.0"
 pyodbc = ">=4.0.34,<6"
 pytest-exasol-itde = ">=0.2.0,<1"
-exasol-toolbox = "^0.20.0"
-
-[tool.poetry.extras]
-pyodbc = ["pyodbc"]
```

Comment:
Can you still to pip install sqlalchemy[pyodbc] after removing this?

Reply:
No, thanks for catching this. Unfortunately, it had gotten lost in a few changes -> had tried to adapt to the newer format `[project.optional-dependencies]`, but it leading to `dependencies` like sqlalchemy being removed from the `poetry.lock`.